app/models/ems_refresh/refreshers/ems_refresher_mixin.rb

```diff
@@ -43,6 +47,7 @@ def refresh
         end
 
         _log.info "Refreshing all targets...Complete"
+        raise PartialRefreshError, partial_refresh_errors.join(', ') if partial_refresh_errors.any?
```

Comment:
I only see the error message being logged [here](https://github.com/ManageIQ/manageiq/blob/95dca49ce029cd4cb11d5bf041093810ca85926e/app/models/miq_queue.rb#L366-L372) and stored in `last_exception` which seems to be used only [here](https://github.com/durandom/manageiq/blob/8befbbdcfb50e977e28bfc04c717a85d8307f164/app/models/vm_or_template.rb#L432)
So I dont think its stored in DB and will only be visible inside the log, where I think its good not to abbreviate messages